hindi/priyanka-chopra

```diff
@@ -0,0 +1 @@
+Priyanka Chopra is an Indian film actress and singer, and the winner of the Miss World pageant of 2000. (https://en.wikipedia.org/wiki/Priyanka_Chopra)
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.